commandLine/src/main.cpp

```diff
@@ -244,7 +246,7 @@ void recursiveUpdate(const fs::path & path, ofTargetPlatform target) {
 	// second check if this is a folder that has src in it
 	if (isGoodProjectPath(path)) {
 		nProjectsUpdated++;
-		auto project = getTargetProject(target);
+//		auto project = getTargetProject(target);
```

Comment:
just curious why this was commented out?

Reply:
updateProject function has the same code inside. so it was just hanging there

Reply:
I should enable unused vars for this project

Reply:
ahh yes - expanding now I see it. Thanks!!